test-it/CallChain.via

```diff
@@ -6,7 +6,7 @@ define(c0 dv(.Int32 5) )
 define(c1 dv(.Int32 1) )
 
 //---------------------------------------------------
-define(videf2 dv(.VirtualInstrument (
+define(videf2%20foo dv(.VirtualInstrument (
```

Comment:
Yes, you can use double quotes here as well.
I'll add a couple of these special characters to the test.  There's no reason to test all of them because I only added decoding, not encoding, and decoding is uniform for all characters.